web_ui_examples/06_machine_learning.md

```diff
@@ -22,16 +22,18 @@ You might recall in the [Data Views guide](03_data_views.md) that we deferred co
 
 ![ML config](fig/51_ml_config.png "ML config")
 
+### Set column types
+
 It is helpful to expand the description ("Show More") and read about the various parameters on this page. On the bottom of that page, each property is listed with its **Descriptor Type** (Categorical, Real, Organic, Inorganic, Alloy), **Parameter Type** (Input, Output, Latent variable, Ignore), and **Values**. If you would like to change a setting, click "edit" next to the corresponding property. You need to have at least one input and at least one output (this is the [supervised learning](https://bigdata-madesimple.com/machine-learning-explained-understanding-supervised-unsupervised-and-reinforcement-learning/) paradigm).
 
-For this tutorial, we'll edit "Property Crystallinity" because we want it as an Input rather than an Output.
+For this tutorial, we'll edit "Property Crystallinity" because we want it as an Input rather than an Output. Specifically, we will use Chemical Formula (Inorganic) and Crystallinity (Categorical) to predict the Band Gap (Real).
 
 ![Set columns](fig/52_set_col_types.png "Set columns")
 
 The above menu will open up, allowing you to change the Variable Type to "Input." Depending on the Descriptor Type, this menu will show different options. *Categorical* descriptors will have all the categories listed for you to include (all are included by default), while *Real* descriptors will have a range of values for you to include. When you're all done—the other properties are fine, though it's a good idea to check each one—click "Okay" to collapse the menu, and finally **Save** at the very top.
 
 ## Model training
-"Training" is the term that refers to a ML model learning the relationships in the data given. Blue progress bars will display at the top of your screen indicating which step it's currently on.
+"Training" is the term that refers to a ML model learning the relationships in the data given. In the background, Citrination trains a **random forest (RF)** model ([Wikipedia](https://en.wikipedia.org/wiki/Random_forest), [Brieman himself](https://www.stat.berkeley.edu/~breiman/RandomForests/cc_home.htm), [~accurate layman example](http://blog.echen.me/2011/03/14/laymans-introduction-to-random-forests/)) to fit the data. Blue progress bars will display at the top of your screen indicating which step of the training process it's currently on.
```

Comment:
I would note that this is a default behavior. "The current default behavior when setting a property as an 'Output' is that we link the Inputs to the Output via a random forest" or similar.

Reply:
Also worth including "Behind the scenes, we build default featurizations based on the input column types (SMILES and Chemical Formula will be featurized differently)." or similar?